giskard_vision/scanner/__init__.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, List, Optional
+
+from .scanner import Scanner
+
+
+def scan(
+    model,
+    dataset,
```

Comment:
for now since we don't have other than landmark detection, I would put `model: FaceLandmarksModelBase`, I would also call the `dataset` -> `dataloader` with `DataIteratorBase` as hint.